.github/workflows/test_stable.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           use-mamba: true
 
       - name: Generate env spec
-        run: pip install tomli && python make_env.py test --py-version ${{ matrix.python-version }}
+        run: pip install tomli && python make_env.py test --py-version ${{ matrix.python-version }}.*
```

Comment:
I don't think this workflow actually tests against a stable release, you still seem to make the env from the same source, so I think this just duplicates the test dev work, though I'm not 100% on that. Maybe we should have a look at this together one of these days.

Reply:
We discussed this in person, it does test against dev and stable in the seperate worfkflows.